manifest.php

```diff
@@ -34,7 +34,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '12.6.3',
+    'version' => '12.6.3.1',
```

Comment:
If possible we should privilege semver usage. As 12.6.4 tag is not used, you should go for 12.6.4

Reply:
i wanted to be 100 sure that everyone understands that this is a back port, and that's why a not so proper semver, 
but i will triple check the 12.6.4 tag and update to that
thank you